tools/outline_proxy_controller/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:20.04
+FROM debian:10
```

Comment:
Ubuntu 20.04 is based on Debian 11. Why doesn't it work? Do you need an older gcc?

Reply:
The issue  #1021 describes the bug in details. Briefly: if you compile with the latest gcc version, it links to system libraries that does not exist on LTS systems.